### PR TITLE
CCv0 | doc: Add crictl e2e cc doc

### PR DESCRIPTION
**Pull request by stevenhorsman**
- Add scripts and documentation to build, configure and test
created a Kata CC unencrypted container using crictl
- Update documentation to better fit the kata documentation requirements

- Fixes: #3510

Signed-off-by: stevenhorsman <steven@uk.ibm.com>

### CODE REVIEW

**Comment by wllenyj** [comment]
Thank you @stevenhorsman 

LGTM

**Comment by fidencio** [comment]
A head's up that may affect your PR.

In the Architecture Committee meeting from January 25th, 2022, the Architecture Committee has agreed on using the "Dismiss stale pull request approvals when new commits are pushed" configuration from GitHub.  It basically means that if your PR has been rebased or updated, the approvals given will be erased.

In order to minimize traumas due to the new approach, please, consider adding a note on the changes done before the rebase / force-push, and also pinging the reviewers for a subsequent round of reviews.

Thanks for your understanding!

Related issue: https://github.com/kata-containers/community/issues/249

**Comment by wainersm** [comment]
@stevenhorsman I will have it. could you give me a few days so that I can also try the instructions? Good opportunity for me to learn...

File: docs/how-to/ccv0.sh
Comment (wainersm) [inline docs/how-to/ccv0.sh:200]: Doesn't it need admin permission to write to `/etc/containerd/config.toml`?

Comment (stevenhorsman) [reply docs/how-to/ccv0.sh:200]: Yes - funny you should mention it. As I comment at the top (at the point I wrote this PR) I'd only ever run it as root for testing (sorry). I have a branch https://github.com/stevenhorsman/kata-containers/tree/ccv0-improvements that I'm going to create a PR for once this is merged that enables it to run as a non-root (but sudo) user, so you could take a look at that instead/as well?

**Comment by stevenhorsman** [comment]
Now the `main` from 4th Jan has merged into `CCv0` I've rebased this, so it needs to be /retest ed

File: docs/how-to/ccv0.sh
Comment (wainersm) [reply docs/how-to/ccv0.sh:200]: Cool. I will have a look and test  #3564

**Review by wainersm — APPROVED** [review]
As we already have an improved version of the script and documentation on #3564, IMO this can be merged as is.

**Pull request merged**
